.serena/memories/retrospective-skill-persistence.md

```diff
@@ -0,0 +1,67 @@
+# Retrospective Skill Persistence Protocol
+
+**Type**: MANDATORY
+**Category**: Agent Workflow
+**Source**: 2025-12-24 User Directive
+
+## Requirement
+
+After EVERY retrospective, you MUST persist extracted skills to Serena memory.
+
+## Process
+
+1. **During retrospective**: Extract skills with SMART validation and atomicity scores
+2. **After retrospective complete**: 
+   - Create individual memory files for each skill (`skill-{domain}-{number}`)
```

Comment:
@rjmurillo-bot This naming format is incorrect. Refer to ADR-017 tiered memory index architecture.